src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,19 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  status: 'Under Construction',
+};
+
+export const categoriesSlice = createSlice({
+  name: 'categoriesSlice',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      const newState = { ...state };
+      newState.status = state.status;
+    },
```

Comment:
- [ ] Kindly make sure to update your state when we call this action your state status must be changed to `Under Construction` because remember at the beginning our state status was empty. Kindly fix that.